components/plurals/tests/fixtures/categories.json

```diff
@@ -3,12 +3,12 @@
     "langid": "ar",
     "plural_type": "Cardinal",
     "categories": [
-      "few",
```

Comment:
Note: You should revert the changes in this file.

Reply:
I don't think I should. This test tests `categories()` not `all()`. And `categories()` are in order because they are used by resolver which benefits from `one` being first. I'll keep it for now. I think at the current state lack of consistency between various orders of iterations on categories is acceptable.